Blackbaud.CustomFx.ConstituentPhotoAnalysis.Catalog/ConstituentPhotoAnalysis.BusinessProcess.cs

```diff
@@ -0,0 +1,179 @@
+using Blackbaud.AppFx.Server;
+using Blackbaud.AppFx.Server.AppCatalog;
+using Blackbaud.CustomFx.ConstituentPhotoAnalysis.SPWrapConstituentPhotoAnalysis;
+using Microsoft.Azure.CognitiveServices.Vision.Face;
+using Microsoft.Azure.CognitiveServices.Vision.Face.Models;
+using System.Data.SqlClient;
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Blackbaud.CustomFx.ConstituentPhotoAnalysis.Catalog
+{
+    public sealed class ConstituentPhotoAnalysisBusinessProcess : AppBusinessProcess
+    {
+
+        // You will need your own subscription key
+        private const string subscriptionKey = "<Replace with your key>";
+
+        //You must use the same region as you used to get your subscription
```

Comment:
Optional: All comments have a space separating the // from the comment text except this one.

Reply:
Done.